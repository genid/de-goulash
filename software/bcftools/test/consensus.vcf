##fileformat=VCFv4.2
##FORMAT=<ID=GT,Number=1,Type=String,Description="Genotype">
##reference=file://some/path/human_g1k_v37.fasta
##INFO=<ID=END,Number=1,Type=Integer,Description="End position of the variant described in this record">
##ALT=<ID=DEL,Description="Deletion">
##contig=<ID=1,assembly=b37,length=249250621>
##contig=<ID=2,assembly=b37,length=249250621>
#CHROM	POS	ID	REF	ALT	QUAL	FILTER	INFO	FORMAT	NA001
1	5	.	C	a	.	PASS	.	GT	0/1
1	5	.	C	t	.	PASS	.	GT	0/1
1	7	.	T	a	.	PASS	.	GT	.
1	10	.	G	a	.	PASS	.	GT	0/1
1	12	.	GACA	ga	.	PASS	.	GT	0/1
1	16	.	T	taaa	.	PASS	.	GT	1/1
1	19	.	A	c	.	PASS	.	GT	0/1
1	61	.	C	a	.	PASS	.	GT	0/1
2	61	.	AGAG	aa	.	PASS	.	GT	0/1
2	119	.	AAA	t	.	PASS	.	GT	0/1
2	179	.	G	gacgtacgt	.	PASS	.	GT	0/1
2	200	.	A	<DEL>	.	PASS	END=210	GT	1/0
2	481	.	T	c,a	.	PASS	.	GT	0/2
