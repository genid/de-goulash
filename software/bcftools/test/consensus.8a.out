>1:2-501
TACAAAATATGATAAAATCAAAAAGAACATAACCTACGTATCAACTAAAGTGGTTGTTTG
AAGAAAAGGAAGACTTAAAAAGAGTCAGTACTAACCTACATAATATATACAATGTTCATT
AAATAATAAAATGAGCTCATCATACTTAGGTCATCATAAATATATCTGAAATTCACAAAT
ATTGATCAAATGGTAAAATAGACAAGTAGATTTTAATAGGTTAAACAATTACTGATTCTC
TTGAAAGAATAAATTTAATATGAGACCTATTTCATTATAATGAACTCACAAATTAGAAAC
TTCACACTGGGGGCTGGAGAGATGGCTCAGTAGTTAAGAACACTGACTGCTCTTCTGAAG
GTCCTGAGTTCAAATCCCAGCAACCACATGGTGACTTACAACCATCTGTAATGACATCTG
ATGCCCTCTGGTGTGTCTGAAGACAGCTACAGTGTACTTACATAAAATAATAAATAAATC
TTTAAAAACAAAAAAAAAGAA
>2
GAAGATCTTTTCCTTATTAAGGATCTGAAGCTCTGTAGATTTGTATTCTATTAAACATGG
AAATTAGTGATTTTCCATATTCTTTAAGTCATTTTAGAGTAATGTGTTCTTAAGATTTCA
GAAAAACAAAAACTTGTGCTTTCCTGTTTGAAAAACAAACAGCTGTGGGGAATGGACGTA
CGTTGTCGGGACAGCCTTTTTATAAAATAATGTTGAGGCTTTGATACGTCAAAGTTATAT
TTCAAATGGAATCACTTAGACCTCGTTTCTGAGTGTCAATGGCCATATTGGGGATTTGCT
GCTGCCAATGACAGCACACCCTGGGAATGCCCCAACTACTTACTACAAAGCAGTGTTACA
TGGAGAAGATCTTCAAGAGTCTTTTTGCTAGATCTTTCCTTGGCTTTTGATGTGACTCCT
CTCAATAAAATCCACAGTAATATAGTGAGTGGTCTCCTGCTCCAAACCAGTATTCCAGAC
ACAGTTAATCCAGAC
